internal/application/command.go

```diff
@@ -141,7 +142,12 @@ func readDeviceResource(device models.Device, resourceName string, attributes st
 }
 
 func readDeviceResourcesRegex(device models.Device, regexResourceName string, attributes string, dic *di.Container) (res *dtos.Event, edgexErr errors.EdgeX) {
-	deviceResources, ok := cache.Profiles().DeviceResourcesByRegex(device.ProfileName, regexResourceName)
+	re, err := regexp.CompilePOSIX(regexResourceName)
+	if err != nil {
+		return res, errors.NewCommonEdgeX(errors.KindContractInvalid, "failed to CompilePOSIX resource name", err)
```

Comment:
Why is `res` used for the event, why not event?

Reply:
added edf083397e33d0cd5580647af7401469445553ce to refactor this